packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.test.ts

```diff
@@ -41,56 +37,46 @@ const ALL_APPOINTMENT_IDS = [
   FUTURE_OTHER_HOSPITAL,
   FUTURE_NOT_BOOKED,
 ];
-
 const reset = async () => {
   await deleteAppointmentsByIds(ALL_APPOINTMENT_IDS);
   await deleteAdmin(COORDINATOR_ID);
   await deleteDonor(DONOR_ID_1);
   await deleteDonor(DONOR_ID_2);
 };
-
 beforeAll(reset);
 afterEach(reset);
-
 test("Unauthenticated user throws exception", async () => {
   const action = () => callFunction();
   await expectAsyncThrows(action, "Unauthorized");
 });
 
-test("User that is not admin throws exception", async () => {
+test("User that is not coordinator throws exception", async () => {
```

Comment:
Should you add more tests to this file? GetCoordinatorAppointmentsHandler changed a lot

Reply:
Sorry did not merge the test files correctly, there is now a test that checks donors in groups